tests/helpers.test.js

```diff
@@ -405,20 +402,60 @@ describe('Helper Tests', () => {
         expect(console.error.mock.calls[0][0]).toContain(packages)
       })
     })
-    describe.skip('checkScriptsFolderExist', () => {
-
+    describe('checkScriptsFolderExist', () => {
+      it("Creates scripts and scripts/templates dir if scripts dir doesn't exist", () => { 
+        fs.existsSync.mockReturnValueOnce(false)
+        helpers.mkdirSync = jest.fn()
+        checkScriptsFolderExist()
+        expect(helpers.mkdirSync).toBeCalledTimes(2)
+        expect(helpers.mkdirSync.mock.calls[0][0]).toEqual('scripts')
+        expect(helpers.mkdirSync.mock.calls[1][0]).toEqual('scripts/templates')
+      })
+      it("Creates scripts/templates dir if scripts/templates does not exist", () => {
+        fs.existsSync.mockReturnValueOnce(true).mockReturnValueOnce(false)
+        helpers.mkdirSync = jest.fn()
+        checkScriptsFolderExist()
+        expect(helpers.mkdirSync).toBeCalledTimes(1)
+        expect(helpers.mkdirSync.mock.calls[0][0]).toEqual('scripts/templates')
+      })
     })
-    describe.skip('getCWDName', () => {
-
+    describe('getCWDName', () => {
+      it("gets the current working directory", () => {
+        const spy = jest.spyOn(process, 'cwd').mockImplementation(() => {
+          return 'usr/local/testApp'
+        })
+        getCWDName()
+        expect(spy).toBeCalled()
+        expect(getCWDName()).toEqual('testApp')
+      })
     })
     describe.skip('modifyKnexExistingProject', () => {
 
     })
     describe.skip('appendFile', () => {
 
     })
-    describe.skip('checkIfScriptIsTaken', () => {
-
+    describe('checkIfScriptIsTaken', () => {
+      it("Checks if a script exists in the package.json", () => {
+        fs.readFileSync.mockReturnValue(`{"scripts": {"test": "do something"} }`)
+        checkIfScriptIsTaken()
+        expect(checkIfScriptIsTaken("test")).toBe(true)
+        expect(checkIfScriptIsTaken("start")).toBe(false)
+      })
+      it("Throws an error if package.json doesn't exist", () => {
+        fs.readFileSync.mockReturnValue(false)
+        console.error = jest.fn()
+        checkIfScriptIsTaken("test")
+        expect(console.error).toBeCalled()
+      })
+      it("Throws an error with the err if package.json doesn't exist in development", () => {
+        fs.readFileSync.mockReturnValue(false)
+        console.error = jest.fn()
+        store.env = 'development'
+        checkIfScriptIsTaken("test")
+        expect(console.error).toBeCalled()
+        expect(console.error.mock.calls[0][0]).not.toContain('Error finding test in package.json')
```

Comment:
Noticing you're doing a lot of "not to contain" hmmmmm. Can we rip those out. It just seems weird